### PR TITLE
Milestone1

### PR DESCRIPTION
**Pull request by 1v4n4**
What I did in this milestone:
- Created React app
-  Delete any unnecessary files
-  Added prop-types and big.js 
- Deployed app on Heroku

### CODE REVIEW

File: src/reportWebVitals.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
-const reportWebVitals = onPerfEntry => {
+const reportWebVitals = (onPerfEntry) => {
   if (onPerfEntry && onPerfEntry instanceof Function) {
-    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
       getCLS(onPerfEntry);
       getFID(onPerfEntry);
       getFCP(onPerfEntry);
```
Comment (george-shammar) [inline src/reportWebVitals.js:8]: Good job so far. You've created an app and added the required dependencies like `big.js.` and `prop-types` :+1: 

- [ ] You still have unnecessary files like this in your repo. e.g `logo.svg` and others. Any file that will not be of direct usage in your app should be deleted.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @1v4n4 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: src/reportWebVitals.js (hunk 1 above)
Comment (1v4n4) [reply src/reportWebVitals.js]: Got it!

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,15 @@
-import logo from './logo.svg';
 import './App.css';
+// import PropTypes from 'prop-types';
+// import Big from 'big.js';
```
Comment (rindrajosia) [inline src/App.js]: - [ ] [OPTIONAL]  Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️

- [ ] [REQUIRED] Please remove also App.test.js, .babelrc and setupTests.js as you are not going to use them. When it will be time to test your app, you are going to use another library.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rindrajosia — APPROVED** [review]
Hi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**